### PR TITLE
Add traces to integration tests + suppress one flaky assertion.

### LINKED ISSUE #11604: E2E tests on electron package have multiple false failures.
There are many failures on develop branch, and also it is blocking many respectable PRs.

Also as part of this task:
* Add test for uploading image
* Rename tests, so the tests of GUI only will be integration tests, and electron package tests will be E2E tests.

### PR DESCRIPTION
**Pull request by farmaazon**
### Pull Request Description

Fixes #11604 

Most issues were caused by a problem with Project List flooding the network with its requests - this was fixed on develop.
But one assertion was flaky - it assumed we will see the "real" run result on `write` node, but sometimes it is immediately overwritten by dry run.

But the most important part of this PR is adding traces to Electron packages - it's should be much easier now to debug E2E test failures.

Also renamed the previously misnamed "E2E tests" to "[GUI] integration tests".

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- ~~[ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- ~~[ ] Unit tests have been written where possible.~~
- ~~[ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.~~
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: app/ide-desktop/client/tests/localWorkflow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  // Press `Write once` button.
+  await writeNode.locator('.More').click()
+  await writeNode.getByTestId('recompute').click()
+
+  // Check that the output file is created and contains expected text.
+  try {
+    await expect(writeNode.locator('.TableVisualization')).toContainText(OUTPUT_FILE)
+  } catch {
+    // TODO[ao]
+    // The above check is flaky, because sometimes the additional egine run overrides node output back to "dry run".
```
Comment (somebody1234) [inline app/ide-desktop/client/tests/localWorkflow.spec.ts]: `egine` -> `engine`

**Review by somebody1234 — APPROVED** [review]
approving rename of dashboard files only

**Comment by somebody1234** [comment]
might wanna update codeowners too due to the change of path

**Comment by MrFlashAccount** [comment]
@farmaazon why we run `pw` tests in `unit tests` job? 
CleanShot 2024-11-22 at 17 50 14@2x

**Review by MrFlashAccount — CHANGES_REQUESTED** [review]
See the comment above

**Review by MrFlashAccount — APPROVED** [review]

**Review by vitvakatu — APPROVED** [review]

**Review by AdRiley — APPROVED** [review]

**Pull request merged**